assets/external_tools_meta.json

```diff
@@ -42,14 +56,60 @@
             "data_url": "https://services.healthtech.dtu.dk/services/NetMHCII-2.2/data.tar.gz",
             "data_md5": "11579b61d3bfe13311f7b42fc93b4dd8",
             "binary_name": "netMHCII"
+        },
+        "2.3": {
+            "version": 2.3,
+            "software_md5": "48f118c133990c9458ef82fa23ef901c",
+            "data_url": "https://services.healthtech.dtu.dk/services/NetMHCII-2.3/data.Linux.tar.gz",
+            "data_md5": "675daa3614ab2ffea47751488caffd74",
+            "binary_name": "netMHCII-2.3"
+        }
+    },
+    "netmhcii_darwin": {
+        "2.2": {
+            "version": 2.2,
+            "software_md5": "1dfd0e69346a7e1d6c9b55cfeb70d744",
+            "data_url": "https://services.healthtech.dtu.dk/services/NetMHCII-2.2/data.tar.gz",
+            "data_md5": "11579b61d3bfe13311f7b42fc93b4dd8",
+            "binary_name": "netMHCII"
+        },
+        "2.3": {
+            "version": 2.3,
+            "software_md5": "c944550e4f07a32ac52524a59e5bf89a",
+            "data_url": "https://services.healthtech.dtu.dk/services/NetMHCII-2.3/data.Darwin.tar.gz",
+            "data_md5": "10fd5c94e716f3921d05ec3ffbb919b2",
+            "binary_name": "netMHCII-2.3"
         }
     },
     "netmhciipan": {
-        "3.1": {
-            "version": 3.1,
-            "software_md5": "0962ce799f7a4c9631f8566a55237073",
-            "data_url": "https://services.healthtech.dtu.dk/services/NetMHCIIpan-3.1/data.tar.gz",
-            "data_md5": "f833df245378e60ca6e55748344a36f6",
+        "4.0": {
+            "version": 4.0,
+            "software_md5": "2bbdb95e20e9991be23e80223d216aef",
+            "data_url": "https://services.healthtech.dtu.dk/services/NetMHCIIpan-4.0/data.tar.gz",
+            "data_md5": "ec3032cb66aee20ff0f53ff7f1795f47",
+            "binary_name": "netMHCIIpan"
+        },
+        "4.1": {
+            "version": 4.1,
+            "software_md5": "c70810798334e9f7614899b7ee965a3a",
+            "data_url": "https://services.healthtech.dtu.dk/services/NetMHCIIpan-4.0/data.tar.gz",
+            "data_md5": "ec3032cb66aee20ff0f53ff7f1795f47",
+            "binary_name": "netMHCIIpan"
+        }
+    },
+    "netmhciipan_darwin": {
+        "4.0": {
+            "version": 4.0,
+            "software_md5": "2bbdb95e20e9991be23e80223d216aef",
+            "data_url": "https://services.healthtech.dtu.dk/services/NetMHCIIpan-4.0/data.tar.gz",
+            "data_md5": "ec3032cb66aee20ff0f53ff7f1795f47",
+            "binary_name": "netMHCIIpan"
+        },
+        "4.1": {
```

Comment:
Also for documentation:
NetMHCIIpan 4.1 (Darwin & Linux) provid the data folder within the software tar, however we did not yet work around this problem. I just thought it would be of interest to have it in order to come up with a solution for this specific version

Reply:
Yes, we should definietly check if we can optimise the process for this case.